test_util/installer_api_test.py

```diff
@@ -21,11 +22,11 @@ def __init__(self):
         self.offline_mode = False
 
     def setup_remote(
-            self, tunnel, installer_path, download_url,
+            self, tunnel: Optional[Tunnelled], installer_path, download_url,
```

Comment:
Note to self: Do we even use both of these interfaces? Should probably just make this tunnel only.